tensorflow/contrib/tpu/profiler/pip_package/cloud_tpu_profiler/main.py

```diff
@@ -17,20 +17,27 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
+from absl import flags
 
 import os
 import subprocess
 import sys
 
 import tensorflow as tf
 
-tf.flags.DEFINE_string('service_addr', '',
+flags.DEFINE_string('service_addr', None,
                        'Address of TPU profiler service e.g. localhost:8466')
-tf.flags.DEFINE_string('logdir', '',
-                       'Path of TensorBoard log directory e.g. /tmp/tb_log')
-tf.flags.DEFINE_integer('duration_ms', 2000, 'Duration of tracing in ms.')
-
-FLAGS = tf.flags.FLAGS
+flags.DEFINE_string('logdir', None,
```

Comment:
I see two of the flags being used below. But not logdir, service_addr or duration_ms.
Where are they used?